odk1-src/collect-forms.rst

```diff
@@ -169,6 +169,8 @@ If you are connected to :doc:`an ODK Aggregate server  <collect-connect-aggregat
   - Use the `TRIM <https://support.google.com/docs/answer/3094140?hl=en>`_ function in the google sheets to remove the leading and trailing spaces from the cells.
   - Define empty cell in your tests to be a cell that is either empty or contains a single space.
 
+  Other values will be preceded by an apostrophe. Thanks to that they won't be parsed automatically in google sheets (e.g. strings to dates) and will be stored as-is.
```

Comment:
I think this doesn't make it clear enough why a user should care and what they might do about it. How about "Other values will be preceded by an apostrophe. This prevents Google Sheets from guessing at the data type and applying a format that may not be appropriate (e.g. making ``1940 10 5`` into a date when it actually is the value of a select multiple). Raw values may be used in any kind of computation and the apostrophes only are visible when editing a value. You may choose to manually apply a format for certain rows if desired." If you can think of a way to make it shorter but include all the info, that would be even better.

Reply:
Ok seems fine. Done.